private/src/scripts/modules/navigation.js

```diff
@@ -6,33 +6,35 @@ let subMenus = [];
 // if menu has lost focus inadvertently, restore it
 const setupFocusTrap = () => {
   const lastMenuItem = pageHeader.querySelector('.mobile-menu > ul > li:last-of-type');
-  const menuItemClassList = `.${Array.from(lastMenuItem.classList).join('.')}`;
-  let previousFocus;
+  if ( lastMenuItem ) {
```

Comment:
This is kind of an unrelated change, but I was getting a scripting error since this was undefined in my test setup.